src/app/screens/Receive/index.tsx

```diff
@@ -112,6 +112,21 @@ function Receive() {
     navigate("/receive");
   }
 
+  function handleBack() {
+    if (invoice == null) {
+      navigate(-1);
+    }
+
+    setFormData({
+      amount: "0",
+      description: "",
+      expiration: "",
+    });
+    setPaid(false);
+    setPollingForPayment(false);
+    setInvoice(null);
+  }
+
```

Comment:
We can DRY this up by using a function called
```
setDefaults() {
    setFormData({
      amount: "0",
      description: "",
      expiration: "",
    });
    setPaid(false);
    setPollingForPayment(false);
    setInvoice(null);
}

Reply:
Will do!